canary-goerli-1.env

```diff
@@ -0,0 +1,15 @@
+CLUSTER_NAME=canary-goerli-1
+NETWORK=goerli
+NODES=7
+NUM_VALIDATORS=1
+CHARON_VERSIONS=latest,latest,latest,latest,latest,latest,latest
+TEKU_VERSION=23.3.0
+lIGTHTHOUSE_VERSION=v3.5.1
+LODESTAR_VERSION=v1.9.2
+VC_TYPES=2,2,2,2,2,2,2
+NIMBUS_VERSION=multiarch-v23.5.1
+BEACON_NODE_ENDPOINTS=http://lighthouse-nethermind-goerli-2.bn-lighthouse-nethermind-goerli-2.svc.cluster.local:5052
+MONITORING_TOKEN="eyJrIjoiYmZmYTU0NzM5NDllYmZkMTE5MTc2NDI4MWM0Y2YxYWFlNGQzYzE3MSIsIm4iOiJvYm9sLWRldiIsImlkIjo2NDg1MDF9"
```

Comment:
@sugh01 GitHub security checks will not allow pushing monitoring and loki tokens. I have two different ideas we can discuss during our call.

Reply:
i will remove this branch.